### PR TITLE
[deps] Update jackson-databind to 2.9.9.3

### PR DESCRIPTION
**Pull request by olivielpeau**
Updates `jackson-databind` to 2.9.9.3

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,6 +43,7 @@
         <jcommander.version>1.35</jcommander.version>
         <log4j.version>1.2.17</log4j.version>
         <jackson.version>2.9.9</jackson.version>
+        <jackson-databind.version>2.9.9.1</jackson-databind.version>
```
Comment (tylerbenson) [inline pom.xml]: Why have a discrepancy between `jackson.version` and `jackson-databind.version`?

Comment (olivielpeau) [reply pom.xml]: 2.9.9.1 and 2.9.9.2 are "micro-patches" that were only released for `jackson-databind`, see https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.9#micro-patches

File: pom.xml [hunk 2]
```diff
@@ -43,6 +43,7 @@
         <jcommander.version>1.35</jcommander.version>
         <log4j.version>1.2.17</log4j.version>
         <jackson.version>2.9.9</jackson.version>
+        <jackson-databind.version>2.9.9.2</jackson-databind.version>
```
Comment (KSerrania) [inline pom.xml]:
```suggestion
        <jackson-databind.version>2.9.9.3</jackson-databind.version>
```
Not sure at all we're affected by this, but `2.9.9.2` introduced a regression that was fixed today in `2.9.9.3` according to [this](https://github.com/FasterXML/jackson/wiki/Jackson-Release-2.9#micro-patches), maybe we can include it as well while we're updating?

Comment (olivielpeau) [reply pom.xml]: nice find, let me update this then (here's hoping there won't be yet another micro release in a few days 😅 )

Comment (olivielpeau) [reply pom.xml]: done, can you review again @KSerrania?

**Pull request merged**